src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/ConnectionManager.java

```diff
@@ -292,20 +303,32 @@ private Channel createChannel() {
     }
 
     private void waitForConnectionRecovery(ShutdownNotifier notifier) {
-        while (!notifier.isOpen() && !connectionIsClosed.get()) {
+        boolean isClose = isConnectionRecovery(notifier);
+
+        if (isClose) {
+            LOGGER.warn("Start wait for recovery connection");
+        }
+
+        while (isClose) {
             try {
                 Thread.sleep(1);
             } catch (InterruptedException e) {
                 LOGGER.error("Wait for connection recovery was interrupted", e);
                 break;
             }
+
+            isClose = isConnectionRecovery(notifier);
         }
```

Comment:
I think we need to write to the log when we stop waiting for connection recovery. Something like this (if you apply the previous suggestion):
```java
if (isClose) { // we have to wait for connection recovery
   LOGGER.info("Stop waiting for connection recovery")
}
```